### PR TITLE
feat: create container style prop

### PR DESCRIPTION
**Pull request by djorkaeffalexandre**
## Motivation
With this prop we can do things like that:
Screen Shot 2020-05-25 at 14 43 34

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -156,6 +158,10 @@ type CommonProps = {
    * Reference to FlatList, ScrollView or SectionList in order to execute its imperative methods.
    */
   innerRef: RefObject<FlatList | ScrollView | SectionList>;
+  /*
+   * Style to be applied handling the handle and the content.
```
Comment (rgommezz) [inline src/index.tsx:162]: "Style to be applied to the container"

File: src/index.tsx [hunk 2]
```diff
@@ -156,6 +158,10 @@ type CommonProps = {
...
+   */
+  containerStyle: StyleProp<ViewStyle>;
```
Comment (rgommezz) [inline src/index.tsx:164]: This property should be optional

**Comment by djorkaeffalexandre** [comment]
@rgommezz done! Thanks.

File: src/index.tsx [hunk 3]
```diff
@@ -606,6 +613,7 @@ export class ScrollBottomSheet<T extends any> extends Component<Props<T>> {
         style={[
           StyleSheet.absoluteFillObject,
           // @ts-ignore
+          containerStyle,
```
Comment (rgommezz) [inline src/index.tsx:616]:
```suggestion
          containerStyle,
          // @ts-ignore
```

**Pull request merged**